<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImpedanceMatchDialog</class>
 <widget class="QDialog" name="ImpedanceMatchDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Impedance Matching</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Unmatched Impedance Z</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="cSource">
            <item>
             <property name="text">
              <string>Custom</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="rbSeries">
              <property name="text">
               <string>Series</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">zGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbParallel">
              <property name="text">
               <string>Parallel</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">zGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Real:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="SIUnitEdit" name="zReal"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Imag:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="SIUnitEdit" name="zImag"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="frequencyLabel">
              <property name="text">
               <string>Frequency:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="SIUnitEdit" name="zFreq"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Matched Impedance</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Real:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="SIUnitEdit" name="mReal">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Imag:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="SIUnitEdit" name="mImag">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Return loss:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="SIUnitEdit" name="mLoss">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QComboBox" name="cMatchType">
          <item>
           <property name="text">
            <string>Series C - Parallel L</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Parallel C - Series L</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Inductor</string>
              </property>
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QRadioButton" name="lIdeal">
                 <property name="text">
                  <string>Ideal</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">lGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QRadioButton" name="lE24">
                 <property name="text">
                  <string>E24</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">lGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QRadioButton" name="lE6">
                 <property name="text">
                  <string>E6</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">lGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QRadioButton" name="lE48">
                 <property name="text">
                  <string>E48</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">lGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QRadioButton" name="lE12">
                 <property name="text">
                  <string>E12</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">lGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QRadioButton" name="lE96">
                 <property name="text">
                  <string>E96</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">lGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="lL">
                <property name="text">
                 <string>L:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="SIUnitEdit" name="lValue">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Capacitor</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="0">
                <widget class="QRadioButton" name="cE6">
                 <property name="text">
                  <string>E6</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">cGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QRadioButton" name="cE12">
                 <property name="text">
                  <string>E12</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">cGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QRadioButton" name="cIdeal">
                 <property name="text">
                  <string>Ideal</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">cGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QRadioButton" name="cE24">
                 <property name="text">
                  <string>E24</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">cGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QRadioButton" name="cE48">
                 <property name="text">
                  <string>E48</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">cGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QRadioButton" name="cE96">
                 <property name="text">
                  <string>E96</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">cGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="lC">
                <property name="text">
                 <string>C:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="SIUnitEdit" name="cValue">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="Image">
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIUnitEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets/siunitedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImpedanceMatchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImpedanceMatchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="lGroup"/>
  <buttongroup name="zGroup"/>
  <buttongroup name="cGroup"/>
 </buttongroups>
</ui>
